src/socketio.quiver.js

```diff
@@ -40,6 +43,19 @@ function QuiverSocialProvider(dispatchEvent) {
   this.dispatchEvent = dispatchEvent;
   this.storage = freedom['core.storage']();
 
+  /** @private @const {!FreedomCrypto} */
+  this.crypto_ = freedom['core.crypto']();
+
+  /** @private @const {!FreedomPgp} */
+  this.pgp_ = freedom['pgp-e2e']();
+
+  /** @type {!Promise<string>} */
+  this.getPubKey_ = this.pgp_.setup('', '<quiver>').then(function() {
```

Comment:
Based on our discussion, this should be ```<uproxy>``` so that we have one key we use across all social networks.  If you prefer we can pass this in as a new login option so that it doesn't have to be hard-coded here